test/README.md

```diff
@@ -211,7 +211,8 @@ Loadbalancer to a NodePort. The external address of such a NodePort is usually
 not easily obtained within the cluster automatically, but can be provided from
 the outside through the `--ingressendpoint` flag. For a minikube setup for
 example, you'd want to run tests against the default `ingressgateway` (port
-31380) running on the minikube node:
+
+31380. running on the minikube node:
```

Comment:
This is broken 🤔

Reply:
Yeah, I need to find time to debug this.  `prettier.io` seems to have a fair number of this kind of bug.